docs/section-2-using-parsers.md

```diff
@@ -629,18 +629,36 @@ The restrictions placed on a pattern by an anchor operator ignore anonymous node
 
 #### Predicates
 
-You can also specify arbitrary metadata and conditions associated with a pattern by adding _predicate_ S-expressions anywhere within your pattern. Predicate S-expressions start with a _predicate name_ beginning with a `#` character. After that, they can contain an arbitrary number of `@`-prefixed capture names or strings.
+You can also specify arbitrary metadata and conditions associated with a pattern
+by adding _predicate_ S-expressions anywhere within your pattern. Predicate S-expressions
+start with a _predicate name_ beginning with a `#` character. After that, they can
+contain an arbitrary number of `@`-prefixed capture names or strings.
 
-For example, this pattern would match identifier whose names is written in `SCREAMING_SNAKE_CASE`:
+Tree-Sitter's CLI supports the following predicates by default:
+
+##### eq?, not-eq?, any-eq?, any-not-eq?
+
+This family of predicates allows you to match against a single capture or string
+value.
```

Comment:
It says about single capture or string, but in a bellow example there are two strings.

Reply:
Fixed